Resources/Prototypes/Entities/Objects/Weapons/Guns/turrets.yml

```
@@ -73,52 +73,81 @@
         - trigger:
             !type:DamageTrigger
-            damage: 300
+            damage: 90 # Rather than fully smash the turret at half health, make an indication that its taking damage.
+          # TODO: Construction graph
+          behaviors:
+            - !type:PlaySoundBehavior
+              sound:
+                collection: MetalGlassBreak
+        - trigger:
+            !type:DamageTrigger
+            damage: 50 # Helps indicate that the turrets arent indestructible.
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)